use-cases/inferencing/serving-with-vllm/batch-inference/src/validate_fine_tuned_model.py

```diff
@@ -0,0 +1,179 @@
+import os
+import requests
+import json
+import pandas as pd
+import logging.config
+from datasets import load_from_disk
+from google.cloud import storage
+
+
+logging.config.fileConfig("logging.conf")
+logger = logging.getLogger("modeleval")
```

Comment:
is there a reason why we are copying code from model-eval? will this code be any different? otherwise we now have the two to maintain

can we not use the model-eval image?

Reply:
I changed this code snippet to reflect prediction class only.Please review new updates.